enterprise/backend/test/metabase_enterprise/sandbox/test_util.clj

```diff
@@ -90,7 +90,7 @@
 (defmacro with-gtaps-for-user
   "Like `with-gtaps`, but for an arbitrary User. `test-user-name-or-user-id` can be a predefined test user e.g. `:rasta`
   or an arbitrary User ID."
-  {:style/indent 2}
+  {:style/indent :defn}
```

Comment:
The correct spec for this is actually `:defn`, because if you don't include the user or GTAPs on the very first line it should not affect the indentation of the `body` of the macro